src/inverseRobot.js

```diff
@@ -6,8 +6,27 @@
  * @return {object}
 */
 
+/**
+ * @param {Object} robot
+ *
+ * @returns {Object|null}
+ */
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (invertedRobot[value] !== undefined
+
+       || Object.values(invertedRobot).includes(value)) {
+      return null;
+    }
```

Comment:
```suggestion
    if (invertedRobot[value] || Object.values(invertedRobot).includes(value)) {
      return null;
    }
```

Actually the second condition looks redundant